mockgcp/mockcompute/regionaloperationsv1.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mockcompute
+
+import (
+	"context"
+
+	pb "github.com/GoogleCloudPlatform/k8s-config-connector/mockgcp/generated/mockgcp/cloud/compute/v1"
+	"k8s.io/klog/v2"
+)
+
+type RegionalOperationsV1 struct {
+	*MockService
+	pb.UnimplementedRegionOperationsServer
+}
+
+func (s *RegionalOperationsV1) Get(ctx context.Context, req *pb.GetRegionOperationRequest) (*pb.Operation, error) {
+	log := klog.FromContext(ctx)
+
+	fqn := s.regionalOperationFQN(req.Project, req.Region, req.Operation)
+	lro, err := s.getOperation(ctx, fqn)
+	if err != nil {
+		log.Info("operation not found", "fqn", fqn)
```

Comment:
nit: same I do not think we need this info message

Reply:
Thanks - removed!